apps/i3wm/i3wm.talon

```diff
@@ -2,8 +2,8 @@
 os: linux
 tag: user.i3wm
 -
-port <number_small>: user.system_command("i3-msg workspace {number_small}")
-port ten: user.system_command("i3-msg workspace 10")
+port <number_small>: user.system_command("i3-msg workspace number {number_small}")
+port ten: user.system_command("i3-msg workspace number 10")
```

Comment:
why is this here?

Reply:
I don't remember anymore. I think it was solving some weird recognition problem from long ago.  I don't use port anymore on mine. Probably can be removed.